### PR TITLE
don't write #form to markdown

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?225093

@orangejenny

### CODE REVIEW

File: tests/markdown.js [hunk 1]
```diff
@@ -10,10 +11,12 @@ define([
     'text!static/markdown/simple-markdown-no-chars.xml',
     'text!static/markdown/no-markdown.xml',
     'text!static/markdown/no-markdown-stars.xml',
-    'text!static/markdown/explicit-no-markdown.xml'
+    'text!static/markdown/explicit-no-markdown.xml',
+    'text!static/markdown/markdown-output-value.xml'
 ], function (
     options,
     util,
+    vellum_util,
```
Comment (millerdev) [inline tests/markdown.js]: should be `vellumUtil`

Comment (emord) [inline tests/markdown.js]: Looks like we use both conventions. tests/widgets uses vellumUtil but databrowser and javaRosa use vellum_util. Should `vellumUtil` be the canonical form?

Comment (millerdev) [inline tests/markdown.js]: yes, to be consistent with the JS naming convention

Comment (millerdev) [inline tests/markdown.js]: can rename them all in a separate PR if you want

**Comment by orangejenny** [comment]
looks good, and @millerdev 's comment isn't blocking

**Pull request merged**